src/sdk_codes.ts

```diff
@@ -0,0 +1,171 @@
+import { Logger } from './log';
+import { Target } from './types';
+
+interface SDKCodeMessages {
+  [key: number]: string;
+}
+
+const sdkCodes: SDKCodeMessages = {
+  // SDK_INIT_1xxx
+  1000: 'The SDK has successfully initialized',
+  1001: 'The SDK has failed to initialize due to an authentication error - defaults will be served',
+  1002: 'The SDK has failed to initialize due to a missing or empty API key - defaults will be served',
+  // SDK_AUTH_2xxx
+  2000: 'Authentication was successful',
+  2001: 'Authentication failed with a non-recoverable error',
+  2002: 'Authentication attempt',
```

Comment:
It's super not obvious here that this is an error state

Reply:
Changed to Authentication attempt failed